merkle/log_proofs_test.go

```diff
@@ -297,11 +297,12 @@ func TestCalcConsistencyProofNodeAddressesBadInputs(t *testing.T) {
 		{size1: 0, size2: 0},
 		{size1: 9, size2: 8},
 	} {
-		_, err := CalcConsistencyProofNodeAddresses(tc.size1, tc.size2, tc.size2)
-
-		if err == nil {
-			t.Fatalf("consistency path calculation accepted bad input: %v", tc)
-		}
+		t.Run(fmt.Sprintf("%d:%d", tc.size1, tc.size2), func(t *testing.T) {
```

Comment:
Could combine these two tests into one.

Reply:
Yeah, this also applies to similar inclusion proof tests. I will do both in a follow-up PR.